### PR TITLE
Update contrib guidelines for creating Github issues

### LINKED ISSUE #700: [DOC] Add more info to contrib guidelines for using Github issues
## Issue description

The section on using issues in the contrib guidelines is very brief.

## Problem location

https://axway-open-docs.netlify.app/docs/contribution_guidelines/#create-an-issue-on-github

## Suggestions for a fix

Add more information about using this option when you want to contribute but don't have the knowledge to do so. Add information about what happens when you submit an issue, e.g. what happens next depending on if it's a simple fix or if it needs R&D, prod mgmt involved.

### PR DESCRIPTION
**Pull request by andreamussap**
Fixes #700 

## Describe the changes

Add more information about using this option when you want to contribute but don't have the knowledge to do so. Add information about what happens when you submit an issue, e.g. what happens next depending on if it's a simple fix or if it needs R&D, prod mgmt involved.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [x] Does not have conflicts with the base branch
* [x] Is clear and complete
* [x] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [x] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [x] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [x] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [x] Does not contain typos, grammatical errors, etc.
* [x] Does not expose any sensitive information
* [x] Passes the Markdown linter rules (automated via CI check)
* [x] Does not contain broken links
* [x] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

**Comment by andreamussap** [comment]
@alexearnshaw please have a look and let me know your fback. Also should I ask Gian/ J Bermingham to review it?

File: content/en/docs/contribution_guidelines/_index.md [hunk 1]
```diff
@@ -153,9 +153,46 @@ When the review is finished, the reviewer will merge the pull request and publis
 
 ## Create an issue on GitHub
 
-Create an issue to inform us about a problem in the documentation. Provide as much detail as you can, for example, if information is missing, contains an error, is not clear, contains a broken link, and so on. The more details you provide, the more helpful the issue, and the faster we can prioritize and fix it.
+If you spot an issue in the documentation but you don't know how to fix it, you can create an issue on Axway-Open-Docs GitHub project to inform about the problem, and provide as much detail as you can, for example:
```
Comment (alexearnshaw) [inline content/en/docs/contribution_guidelines/_index.md:156]: => If you find an issue or problem in the documentation that you don't know how to fix, create an issue in Axway-Open-Docs GitHub project to report it to us.

(new paragraph)
When creating a documentation issue, provide as much detail as you can. For example:
* Is it missing important information?
* Does it contain errors or inaccuracies?
* Is it unclear or hard to follow?
* Is it missing examples?
* Does it contain a broken link or broken image link?

(I think these sentences are a bit clearer and I think asking users a list of questions to think about is more interactive than just listing them out)

File: content/en/docs/contribution_guidelines/_index.md [hunk 2]
```diff
@@ -153,9 +153,46 @@ When the review is finished, the reviewer will merge the pull request and publis
...
 
-To create an issue just click **Create documentation issue** on the upper right corner of the page, and add the relevant information.
+* Information is missing
+* Contains an error
+* It's not clear
+* Needs and example
+* Contains a broken link, and so on.
+
+The more details you provide the more helpful the issue, and the faster we can prioritize and fix it.
```
Comment (alexearnshaw) [inline content/en/docs/contribution_guidelines/_index.md:164]: I think the comma is needed after **provide**...if I was speaking this sentence that's where I would pause (and again after **issu**e). But as I don't know what rule of English explains why a comma should or shouldn't be there, I'd like to hear your reasoning  :)

File: content/en/docs/contribution_guidelines/_index.md [hunk 3]
```diff
@@ -153,9 +153,46 @@ When the review is finished, the reviewer will merge the pull request and publis
...
+
+To create an issue from the documentation:
```
Comment (alexearnshaw) [inline content/en/docs/contribution_guidelines/_index.md:166]: => To create a documentation issue:

(**from the documentation** makes it sound like you're going to use the documentation to create an issue)

File: content/en/docs/contribution_guidelines/_index.md [hunk 4]
```diff
@@ -153,9 +153,46 @@ When the review is finished, the reviewer will merge the pull request and publis
...
+
+1. Click **Create documentation issue** on the upper right corner of any page.
+
+    You will be redirected to the GitHub sign in page.
```
Comment (alexearnshaw) [inline content/en/docs/contribution_guidelines/_index.md:170]: Is this needed? We don't spell that out in the other sections, and it only happens if you're not already signed in. Remove Step 2 for the same reason. 
If you think this is important, maybe just have an explanation in step 1 e.g. 

1. Click **Create documentation issue** on the upper right corner of any page. You are redirected to GitHub and asked to sign in with your GitHub account.
2. Add a title...

File: content/en/docs/contribution_guidelines/_index.md [hunk 5]
```diff
@@ -153,9 +153,46 @@ When the review is finished, the reviewer will merge the pull request and publis
...
+2. Sign in to GitHub.
+3. Add a title to the issue.
+
+    The title of the issue is completed by default with the name of the page where you were in, but you can change or complement it to make it more helpful.
```
Comment (alexearnshaw) [inline content/en/docs/contribution_guidelines/_index.md:174]: =>The title of the issue is completed by default with the name of the page, but you can edit it to make it more helpful.
(more minimal)

File: content/en/docs/contribution_guidelines/_index.md [hunk 6]
```diff
@@ -153,9 +153,46 @@ When the review is finished, the reviewer will merge the pull request and publis
...
+4. Provide a detailed information about the problem.
```
Comment (alexearnshaw) [inline content/en/docs/contribution_guidelines/_index.md:175]: => Provide detailed information about the problem. A template is provided.
(I don't think it's worth to duplicate the template here as it might change)

File: content/en/docs/contribution_guidelines/_index.md [hunk 7]
```diff
@@ -153,9 +153,46 @@ When the review is finished, the reviewer will merge the pull request and publis
...
+
+    <!--Give the URL of the page or section that contains the problem.-->
+
+    ## Suggestions for a fix
+
+    <!--If you have specific ideas about how we can fix this, let us know. -->
+    ```
+5. Click **Submit new issue**.
+
+Once the issue is created, GitHub will send an email notification to you and to the maintainers of the project informing about the new issue. You can also find your issue on the Axway-Open-Docs [list of issues](https://github.com/Axway/axway-open-docs/issues).
```
Comment (alexearnshaw) [inline content/en/docs/contribution_guidelines/_index.md:193]: Once = > Replace with when or after (style guide)
GitHub will send => GitHub sends (avoid future tense, style guide)
informing about the new issue => remove this, it's not really needed and is clear enough without it

File: content/en/docs/contribution_guidelines/_index.md [hunk 8]
```diff
@@ -153,9 +153,46 @@ When the review is finished, the reviewer will merge the pull request and publis
...
+
+The maintainers will review the issue and, either fix it if possible or involve the R&D subject matter experts if it needs further investigation. If the R&D needs to be involved, a Jira ticket will be created to track the issue internally. It is important that you monitor your GitHub email notifications to follow-up with the updates of your issue.
```
Comment (alexearnshaw) [inline content/en/docs/contribution_guidelines/_index.md:195]: Suggest breaking this out a bit:
The maintainers will review the issue. If the problem is straightforward the maintainers will fix it as soon as possible. If the issue needs further investigation or clarification from a subject matter expert, an internal JIRA ticket will be created and added to the product backlog.

(new para)
=>It is important that you monitor your GitHub email notifications to keep informed of any updates to your issue. (just reads better imho)

File: content/en/docs/contribution_guidelines/_index.md (hunk 2 above)
Comment (andreamussap) [reply content/en/docs/contribution_guidelines/_index.md:164]: TBH, I had the same 'feeling' as you but I continued with the rest of the file and forgot to recheck this. And yes, there should be a comma, although I'll certanly forget how to explain the grammar rule to someone in the future :) See '[Comparative Clauses - The more, the more](https://www.grammar-quizzes.com/themore.html)'

File: content/en/docs/contribution_guidelines/_index.md (hunk 6 above)
Comment (andreamussap) [reply content/en/docs/contribution_guidelines/_index.md:175]: @alexearnshaw  I'm not duplicating. When you click 'Create an issue' from the docs, the template is not filled in, just when you go to the Git project and create an issue from there. 

Can you configure the Create Issue from the docs to fill in the template?

Comment (alexearnshaw) [reply content/en/docs/contribution_guidelines/_index.md:175]: Ah sorry! Didn't realise that. I'm checking if this is possible in PR https://github.com/Axway/axway-open-docs/pull/727
